eth/db/storage.py

```diff
@@ -129,9 +56,9 @@ class StorageLookup(BaseDB):
     """
     logger = cast(ExtendedDebugLogger, logging.getLogger("eth.db.storage.StorageLookup"))
 
-    def __init__(self, db: BaseDB, account_db: BaseAccountDB, address: Address) -> None:
+    def __init__(self, db: BaseDB, storage_root: Hash32, address: Address) -> None:
         self._db = db
-        self._account_db = account_db
+        self._storage_root = storage_root
```

Comment:
Now that `_storage_root` is constant, I think that `_storage_root` can be renamed to `_starting_root_hash`, and some of the manipulation of `_starting_root_hash` can be dropped.